runtime-modules/proposals/codex/src/lib.rs

```diff
@@ -835,125 +785,25 @@ impl<T: Trait> Module<T> {
             params.title.clone(),
         )?;
 
-        let proposal_id = <proposals_engine::Module<T>>::create_proposal(
+        let proposal_creation_params = ProposalCreationParameters {
             account_id,
-            params.member_id,
-            params.proposal_parameters,
-            params.title,
-            params.description,
-            params.stake_balance,
-            params.proposal_code,
-        )?;
+            proposer_id: params.member_id,
+            proposal_parameters: params.proposal_parameters,
+            title: params.title,
+            description: params.description,
+            stake_balance: params.stake_balance,
+            encoded_dispatchable_call_code: params.proposal_code,
+            exact_execution_block: None,
```

Comment:
Why is this `None` ?

Reply:
This is a bug. Thanks for catching, forgot to remove the intermediate stub.